ouroboros-consensus/test-storage/Test/Ouroboros/Storage/ImmutableDB/StateMachine.hs

```diff
@@ -743,12 +767,13 @@ postcondition model cmdErr resp =
 
 -- | Environment to run commands against the real ImmutableDB implementation.
 data ImmutableDBEnv h = ImmutableDBEnv
-  { varErrors :: StrictTVar IO Errors
-  , varNextId :: StrictTVar IO Id
-  , registry  :: ResourceRegistry IO
-  , hasFS     :: HasFS IO h
-  , db        :: ImmutableDB    Hash IO
-  , internal  :: ImmDB.Internal Hash IO
+  { varErrors  :: StrictTVar IO Errors
+  , varNextId  :: StrictTVar IO Id
+  , varCurSlot :: StrictTVar IO SlotNo
```

Comment:
Always `maxBound`, unless executing `ReopenInThePast`. Mention it again in the case for `Reopen` in `run`.